packages/service-library/tests/test_archiving_utils.py

```diff
@@ -171,20 +160,47 @@ def _relative_path(input_path: Path) -> Path:
         )
 
 
-# end utils
+def assert_unarchived_paths(
+    unarchived_paths: Set[Path], src_dir: Path, dst_dir: Path, is_saved_as_relpath: bool
+):
+    is_file_or_emptydir = lambda p: p.is_file() or (p.is_dir() and not any(p.glob("*")))
+
+    # all unarchivedare under dst_dir
+    assert all(dst_dir in f.parents for f in unarchived_paths)
+
+    # can be also checked with strings
+    assert all(str(f).startswith(str(dst_dir)) for f in unarchived_paths)
```

Comment:
maybe use path instead of f?

Reply:
line 172 is checking exactly the same as line 169 but "the old way" . In fact line 169 is more reliable ... I was just comparing